README.md

```diff
@@ -10,7 +10,7 @@ This mono repo combines four packages:
  - A [stencil web component](packages/web-component/) for visualizing read alongs,
  - An [Angular Library](packages/ngx-web-component/) that wraps the stencil web component,
  - A [demo web application](packages/angular-demo/) to show how to use the angular library in an Angular web application.
-- The [Studio-Web](packages/Studio-Web/) application for creating ReadAlongs
+- The [studio-web](packages/studio-web/) application for creating ReadAlongs
```

Comment:
should the name of the link still be Camel-Kebab-Cased?

Reply:
Right, yes, it should. I caught (and kept) some of them, but not this one.